README.md

```diff
@@ -68,6 +68,17 @@ The Inter-Blockchain Communication protocol (IBC) allows blockchains to talk to
 
 For an overview of upcoming changes to ibc-go take a look at the [roadmap](./docs/roadmap/roadmap.md).
 
+## Ecosystem
+
+Discover what apps, middleware and light clients are developed by other awesome teams in the ecosystem:
+
+|Repository|Type|
+|----------|----|
+|[icq](https://github.com/strangelove-ventures/ibc-go/tree/feature/icq_implementation/modules/apps/icq)|`app`|
+|[interchain-queries](https://github.com/ingenuity-build/interchain-queries)|`app`|
+|[query](https://github.com/defund-labs/defund/tree/main/x/query)|`app`|
+|[packet-forward-middleware](https://github.com/strangelove-ventures/packet-forward-middleware)|`middleware`|
```

Comment:
I started with this list. Happy to add any more that people know of. I haven't add NFT or beefy yet, since there's no repository available yet. Should I add maybe at least the text without the link?

Other columns to add to the table?

Reply:
I think it would be great to add a short snippet of what the application is actually doing, as well as a specific column for the implementing entities (maybe links there too).

there also a difference between applications and middleware, maybe it would make sense to indicate these also differently in this registry?

Reply:
> I think it would be great to add a short snippet of what the application is actually doing, as well as a specific column for the implementing entities (maybe links there too).

Sounds good! I will add these.

> there also a difference between applications and middleware, maybe it would make sense to indicate these also differently in this registry?

The table has a column `Type` which at the moment takes the values `app` and `middleware` for the entries that are right now in the table. Would that be enough?

Reply:
ah, i meant more like, maybe indicate what an "app" is compared to "middleware" with links or a short desciption